EasyCommands/CommandParsers/ParameterParsers.cs

```diff
@@ -97,6 +97,12 @@ partial class Program {
         static String[] colorKeywords = { "color" };
         static String[] textKeywords = { "text", "message" };
 
+        static String[] blinkIntervalKeywords = { "blinkinterval", "blinkInterval", "interval" };
```

Comment:
Given we included "interval", we might consider having the property name be "intervl" in case we can use this across other block types